sig-scalability/README.md

```diff
@@ -95,17 +95,18 @@ Also do not hesitate to contact those SIG members for status update:
 * Wojciech Tyczynski (**[@wojtek-t](https://github.com/wojtek-t)**), Google
 
 ## Upcoming 2020 Meeting Dates
-   * 01/30
-   * 02/13
-   * 02/27
-   * 03/12
-   * 03/26
-   * 04/09
-   * 04/23
-   * 05/07
-   * 05/21
-   * 06/04
-   * 06/18
+   * Jan - 30
+   * Feb - 13, 27
+   * Mar - 12, 26
+   * Apr - 09, 23
+   * May - 07, 21
+   * Jun - 04, 18
+   * Jul - 02, 16, 30
+   * Aug - 13, 27
+   * Sep - 10, 24
+   * Oct - 08, 22
+   * Nov - 05, 19
+   * Dec - 03, 17
```

Comment:
Given what is happening - I think we may consider doing the weekly (or alternatively making them hour-long).
I wanted to touch it today, but had to leave earlier. But we should keep that in mind.
@shyamjvs @mm4tt - thoughts?

Reply:
Yeah seems like that might be needed for a while. I'm voting for weekly rather than hour-long given it can help new contributors faster feedback cycle. Matt - Wdyt?

Reply:
I also vote for weekly.

Reply:
I've looked in the timezone converter http://www.thetimezoneconverter.com/?t=17:30&tz=UTC and it seems to convert fine to Warsaw and Pacific time.

I'm not sure though if will convert fine event after DST ends (don't know how to verify it there).

Reply:
SGTM, let's change it to weekly and announce on Slack after this gets merged

Reply:
Done. I've made it weekly for April and updated meeting dates. For further months we can decide later as needed. SG?

Reply:
LGTM